org.eclipse.winery.lsp/src/main/java/org/eclipse/winery/lsp/Server/ServerAPI/API/context/LSContext.java

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
All classes need to get headers (see https://www.eclipse.org/projects/handbook/#ip-copyright-headers for background)

Some of your classes have - please check that all classes have!